app/models/comment.rb

```diff
@@ -17,6 +17,11 @@ class Comment < ApplicationRecord
   has_many :comments, as: :commentable, dependent: :destroy
 
   scope :ordered, -> { order(created_at: :desc) }
+  scope :last_week, -> { where('updated_at >= ?', 1.week.ago) }
 
   validates :content, presence: true
+
+  def self.count_per_day(day)
```

Comment:
why don't you use scope for this?

Reply:
I thought about this then I was writing this method. And I decide to write a method, because it returns only count  for collection, I thought that scopes - for filtering, ect.